kiva/compat.py

```diff
@@ -0,0 +1,23 @@
+"""
+Compatibility module to help support various versions of PIL and
+Pillow.
+
+"""
+import PIL
+from PIL import Image
+
+PILLOW_VERSION = getattr(PIL, 'PILLOW_VERSION', PIL.VERSION)
+
+
+def piltostring(image):
+    if PILLOW_VERSION.startswith('3'):
+        return image.tobytes()
```

Comment:
Sorry, no coffee on this side either, I saw the `env` in the `.travis.yml` and thought that was being read here. Now I see that it's just controlling the dependencies that get installed for the matrix of tests. Yes, this is fine. What I was suggesting probably would have also worked, though: test for the presence of `tobytes()`, not the absence of `tostring()`.